ui/src/components/ExportFab.js

```diff
@@ -25,10 +34,42 @@ class ExportFab extends React.Component {
           <CloudUpload />
         </Button>
       </Tooltip>
+       <div>
+        <Dialog
```

Comment:
Sorry changed my mind, can you add the Cancel button back in? Thanks.

Reply:
Done